esp-hal/src/rmt.rs

```diff
@@ -1478,72 +1477,40 @@ fn async_interrupt_handler() {
     };
     match channel {
         0 => {
-            <Channel<Async, 0> as TxChannelInternal>::unlisten_interrupt(
-                Event::End | Event::Error,
-            );
-            <Channel<Async, 0> as RxChannelInternal>::unlisten_interrupt(
-                Event::End | Event::Error,
-            );
+            <Channel<Async, 0> as TxChannelInternal>::unlisten_interrupt(Event::End | Event::Error);
```

Comment:
WHY

Reply:
mhhh - the xtask doesn't touch these lines for me but VSCode "Reformat Code" does 🤔 - would be good if they would agree (at least the CI checks don't care - not sure if that is a good or a bad thing)

Reply:
Well the result is better, but the reformatting is soooo unnecessary...

Reply:
oh - it _seems_ that `cargo fmt` (and so does `xtask fmt-packages`) ignores modules which contained in an `unstable_module! { ... }` 😲 - currently I can do "anything" in terms of formatting to e.g. `rmt.rs` and it will stay that way.

When I move the module out of the macro-invocation it gets reformatted (and the same way we see here)